packages/hebao_v1/contracts/stores/SecurityStore.sol

```diff
@@ -184,26 +151,24 @@ contract SecurityStore is DataStore
         uint idx = w.guardianIdx[guardian];
         require(idx > 0, "GUARDIAN_NOT_EXISTS");
         require(
-            w.guardianEffectiveTime[guardian] > 0 &&
-            w.guardianEffectiveTime[guardian] < now,
+            isGuardianPendingAddition(wallet, guardian),
             "NOT_PENDING_ADDITION"
         );
 
         Data.Guardian memory lastGuardian = w.guardians[w.guardians.length - 1];
-
         if (guardian != lastGuardian.addr) {
             w.guardians[idx - 1] = lastGuardian;
             w.guardianIdx[lastGuardian.addr] = idx;
         }
         w.guardians.pop();
         delete w.guardianIdx[guardian];
-        delete w.guardianRemovalEffectiveTime[guardian];
+        delete w.guardianValidUntil[guardian];
```

Comment:
I think this should be `delete w.guardianValidSince[guardian];`.

Reply:
Yes, you are right.